docs/platform/reference/maintenance-downtime.rst

```diff
@@ -0,0 +1,12 @@
+Maintenance downtime window
+===========================
+
+Maintenance window is a time window during which we'll switch to new upgraded virtual machines and point the connection URI DNS name at the new server.
+
+Aiven service upgrades are performed in rolling forward style, which means that new service nodes are first created alongside with the older nodes one at a time, after which the old nodes are retired.
+
+In case of **MySQL®**, **PostgreSQL®** and **Redis®*** the maintenance window usually lasts around 5-10 seconds. The downtime comes from old master stopping itself in a controlled manner and new master executing promotion sequence after this. Once the promotion is complete the old master node starts forwarding requests to the new master node so the service is accessible before DNS updates are propagated, though clients that end up reconnecting to the old master node will see additional disconnection once the old master is permanently retired.
```

Comment:
Are we sure about the timeline here? Or should we be more conservative saying "some seconds" without any fixed limit?